### PR TITLE
Document verbosity increase for gazebo

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
This documents increasing the verbosity in the simulation through `VERBOSE_SIM` when running SITL simulations

Related PR: https://github.com/PX4/Firmware/pull/15817 https://github.com/PX4/Firmware/pull/15720

### CODE REVIEW

File: en/simulation/gazebo.md [hunk 1]
```diff
@@ -375,6 +375,11 @@ The code is available on the [sitl_gazebo repository](https://github.com/px4/sit
 > **Note** The build system enforces the correct GIT submodules, including the simulator.
   It will not overwrite changes in files in the directory.
 
+In case something is not working as expected, you can increase the verbosity of the gazebo simulation through `VERBOSE_SIM`
```
Comment (TSC21) [inline en/simulation/gazebo.md]: Should we specify if this is to `gzserver` or `gzclient` or both?

Comment (hamishwillee) [reply en/simulation/gazebo.md]: @Jaeyoung-Lim 
Thanks. I have moved to its own heading/section. This isn't a bad place at all - I just think it is useful to have all the info about flags in one place, and I think a search is more likely to find this. 

FWIW it this only logs the server or the client we should say that (ie I agree with Nuno). If it does both we can ignore.

File: en/simulation/gazebo.md [hunk 2]
```diff
@@ -367,6 +367,23 @@ gst-launch-1.0  -v udpsrc port=5600 caps='application/x-rtp, media=(string)video
 ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink fps-update-interval=1000 sync=false
 ```
 
+### Verbose Logging
+
```
Comment (hamishwillee) [inline en/simulation/gazebo.md:371]: @Jaeyoung-Lim Please check - and if this logs specifically only part of gazebo (server or client) or any other very specific part, we should identify that part. If it is for general use we can probably leave as is.

Comment (RicardoM17) [reply en/simulation/gazebo.md:371]: @Jaeyoung-Lim wink wink :point_up:

**Comment by hamishwillee** [comment]
@Jaeyoung-Lim I am merging this despite https://github.com/PX4/PX4-Devguide/pull/1088#discussion_r496249197 . This is still useful. Would be good if you could address though.

**Pull request merged**